ivy_tests/test_ivy/helpers/function_testing.py

```diff
@@ -667,6 +685,29 @@ def test_frontend_function(
                 else:
                     assert ret.data is out.data
             assert ret is out
+    elif test_flags.with_copy:
+        assert not isinstance(ret, tuple)
+
+        if test_flags.generate_frontend_arrays:
+            assert _is_frontend_array(ret)
+        else:
+            assert ivy.is_array(ret)
```

Comment:
Why is is this assertion specific to `test_flags.with_copy`?

Reply:
This assertion was only available in `test_flags.inplace` I didn't know whether I should refactor it or add the same assertion to `test_flags.with_copy`.